[Excel] (Custom functions) Update low engagement articles (#5364)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-runtime.md

---
ms.date: 10/22/2025
description: Understand Excel custom functions that don't use a shared runtime and their specific JavaScript-only runtime.
title: JavaScript-only runtime for custom functions
ms.localizationpriority: medium
---

# JavaScript-only runtime for custom functions

> Custom functions that don't use a shared runtime instead use a [JavaScript-only runtime](../testing/runtimes.md#javascript-only-runtime) that's focused on fast calculation and has fewer APIs available.

Reviewer: ```suggestion
Custom functions that don't use a shared runtime rely on a [JavaScript-only runtime](../testing/runtimes.md#javascript-only-runtime). This runtime is optimized for fast calculation but has fewer APIs available.
```

}
```

## Compare with shared runtime

> Need UI integration, richer Office.js objects, or advanced events? Move those functions to a [shared runtime](../testing/runtimes.md#shared-runtime) so they can use those features.

Reviewer 2: I don't understand what a rich Office.js object is, or advanced events (in your custom function?)
